core/proc.rbs

```diff
@@ -262,7 +262,7 @@ class Proc < Object
 
   def []: (*untyped arg0) -> untyped
 
-  def curry: (?Integer arity) -> Proc
+  def curry: (?_ToInt arity) -> Proc
```

Comment:
```suggestion
  def curry: (?int arity) -> Proc
```

`int` looks work too, both is same? Then which one should be preferred? 🤔

Reply:
It's equivalent because `_ToInt` is `int`.

Reply:
Sure!